blocklist-client/tests/tls_checking.rs

```diff
@@ -0,0 +1,15 @@
+use std::time::Duration;
+
+const REQUEST_TIMEOUT: Duration = Duration::from_secs(5);
+
+#[tokio::test]
+async fn check_tls_support() {
+    let resp = reqwest::Client::new()
+        .get("https://google.com")
```

Comment:
We will probably regret this (eg, CI failing because some random runner ban or something).

Reply:
Yeah I agree.